libs/langchain/langchain/vectorstores/redis/schema.py

```diff
@@ -141,25 +144,21 @@ class HNSWVectorField(RedisVectorField):
     m: int = Field(default=16)
     ef_construction: int = Field(default=200)
     ef_runtime: int = Field(default=10)
-    epsilon: float = Field(default=0.8)
```

Comment:
seems like a big change — will this meaningfully change default behavior?

Reply:
This is actually the correct default value from the docs:
https://redis.io/docs/interact/search-and-query/advanced-concepts/vectors/#hnsw

Good news is that it only impacts Redis RangeQueries, which are less commonly used thus far. I think it's best we get this right here moving forward.

Reply:
This was the right thing to do. Either way, I don't believe this will have a huge impact.